dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/CrudControllerAdvice.java

```diff
@@ -363,12 +363,10 @@ public WebMessage handlePotentialDuplicateForbiddenRequest( Exception exception
      * Catches default exception and send back to user, but re-throws internally
      * so it still ends up in server logs.
      */
-    @ExceptionHandler( Exception.class )
     @ResponseBody
+    @ExceptionHandler( Exception.class )
     public WebMessage defaultExceptionHandler( Exception ex )
     {
-        // We print the stacktrace so it shows up in the logs, so we can more
-        // easily understand 500-issues.
```

Comment:
I think we put this comment deliberately so nobody would remove the `ex.printStackTrace();`